include/grpcpp/impl/codegen/proto_utils.h

```diff
@@ -77,6 +77,14 @@ Status GenericDeserialize(ByteBuffer* buffer, grpc::protobuf::Message* msg) {
     return Status(StatusCode::INTERNAL, "No payload");
   }
   Status result = g_core_codegen_interface->ok();
```

Comment:
Let's just use `Status result;` as ok should be the default...

Reply:
Great point, and it'll likely save a call since the empty constructor can be inlined but not the interface call.

Reply:
Done